src/main/java/org/helpberkeley/memberdata/v200/WorkflowBeanV200.java

```diff
@@ -251,20 +252,38 @@ public void setOrders(String orders) {
     }
 
     public String toCSVString() {
-        return consumer + Constants.CSV_SEPARATOR +
-                driver + Constants.CSV_SEPARATOR +
-                name + Constants.CSV_SEPARATOR +
-                userName + Constants.CSV_SEPARATOR +
-                phone + Constants.CSV_SEPARATOR +
-                altPhone + Constants.CSV_SEPARATOR +
-                neighborhood + Constants.CSV_SEPARATOR +
-                city + Constants.CSV_SEPARATOR +
-                address + Constants.CSV_SEPARATOR +
-                condo + Constants.CSV_SEPARATOR +
-                details + Constants.CSV_SEPARATOR +
-                restaurant + Constants.CSV_SEPARATOR +
-                normal + Constants.CSV_SEPARATOR +
-                veggie + Constants.CSV_SEPARATOR +
-                orders;
+        return Exporter.escapeCommas(consumer) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(driver) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(name) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(userName) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(phone) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(altPhone) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(neighborhood) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(city) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(address) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(condo) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(details) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(restaurant) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(normal) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(veggie) + Constants.CSV_SEPARATOR +
+                Exporter.escapeCommas(orders);
+    }
+
+    public String getCSVHeader() {
```

Comment:
In production we do WorkflowParser.auditColumnNames, but we don't actually have the assertion that incoming and outgoing headers are the same. Should I add that?

Reply:
Yes.  It would probably a good assert in WorkflowExporter.updateMemberData() at the very end of the method.  Something like:
```assert incomingHeader.equals(outgoingHeader) : "header mismatch:\n" + incomingHeader + "\n" + outgoingHeader```